docs/reference/cluster/prevalidate-node-removal.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[prevalidate-node-removal-api]]
+=== Prevalidate node removal API
+++++
+<titleabbrev>Prevalidate node removal</titleabbrev>
+++++
+
+NOTE: {cloud-only}
+
+Prevalidate node removal.
+
+[[prevalidate-node-removal-api-request]]
+==== {api-request-title}
+
+`POST /_internal/prevalidate_node_removal/<nodes>`
+
+[[prevalidate-node-removal-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `monitor` or `manage` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+[[prevalidate-node-removal-api-desc]]
+==== {api-description-title}
+
+This API allows validating whether it is safe to remove the specified node(s) from the cluster. In case the cluster has a <<cluster-health,`red` cluster health status>>, this verifies that the removal of the node(s) would not risk removing the last remaining copy of an unassigned shard.
+
+The response includes the overall safety of the removal of the specified nodes, and a detailed response for each node. The node-specific part of the response also includes more details on why removal of that node might not be safe.
+
+The result of the prevalidation is expressed as either `YES`, `NO`, or `UNKNOWN`.
+
+[[prevalidate-node-removal-api-path-params]]
+==== {api-path-parms-title}
+
+`<node>`::
+    (Required, string) Comma-separated list of node IDs, names or external IDs.
```

Comment:
I did not look at the code yet, but it would be nice if we know what field we expect here to disambiguate.

I think we should not support IP etc., those 3 forms you mention are all we need.